playbooks/templates/.env.j2

```diff
@@ -47,42 +55,56 @@ PORTAL_NAME={{ webportal_server_config.portal_name }}
 # PORTAL_MODULES=mj
 # or
 # PORTAL_MODULES=aj
+#
+# REQUIRED: manually defined by user
 PORTAL_MODULES={{ webportal_server_config.portal_modules }}
 
 # Absolute url to the server API.
+#
+# REQUIRED: See top of file 
 SKYNET_SERVER_API=https://{{ server_domain }}
 
 # Sia wallet password
+#
+# REQUIRED: Ansible automatically generated 
 SIA_WALLET_PASSWORD={{ webportal_server_config.sia_wallet_password if (webportal_server_config.sia_wallet_password is defined and webportal_server_config.sia_wallet_password is not none and webportal_server_config.sia_wallet_password != '') else (webportal_server_config.sia_wallet_seed if webportal_server_config.sia_wallet_seed is defined else '') }}
 
-# (optional) The port on which siad is listening, defaults to 9980.
+# The port on which siad is listening, defaults to 9980. 
+#
+# OPTIONAL: manually defined by user, or default will be used
+# TODO: Does this default need to be explicitly set?
```

Comment:
API_PORT is only used by `blocker`, if I'm not mistaken, and it doesn't need it to be defined - it will happily use 9980 otherwise.

Reply:
we should just remove this then.